src/frontend/context/SpråkContext.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+
+import createUseContext from 'constate';
+import { useCookies } from 'react-cookie';
+
+import { onLanguageSelect, setParams } from '@navikt/nav-dekoratoren-moduler';
+
+import { LocaleType } from '../typer/common';
+
+const dekoratorLanguageCookieName = 'decorator-language';
+
+export const [SpråkProvider, useSpråk] = createUseContext(() => {
+    const [cookies] = useCookies([dekoratorLanguageCookieName]);
+    const { [dekoratorLanguageCookieName]: dekoratørSpråk } = cookies;
+
+    const defaultSpråk = (dekoratørSpråk as LocaleType) ?? LocaleType.nb;
+
+    const [valgtLocale, settValgtLocale] = useState<LocaleType>(defaultSpråk);
+
+    useEffect(() => {
+        setParams({
+            language: defaultSpråk,
+            availableLanguages: [
+                { locale: 'nb', handleInApp: true },
+                { locale: 'nn', handleInApp: true },
+                { locale: 'en', handleInApp: true },
+            ],
+        }).then(); // Bryr oss egenlig ikke om hva som skjer etterpå men intellij klager på ignorert promise
```

Comment:
Etter at jeg la inn støtte for å kun vise språkvelgeren på forsiden er jeg faktisk litt usikker på om vi ønsker å sette availableLanguages her også 🤔